lib/sdl/cursors_sdl.cpp

```diff
@@ -29,6 +29,7 @@
 #include <SDL.h>
 
 static CURSOR currentCursor = CURSOR_MAX;
+static bool newCursor = false;
```

Comment:
```suggestion
static CURSOR lastAppliedCursor = CURSOR_MAX;
```
Instead of storing a `bool`, store the `CURSOR` last set via `wzApplyCursor`, thus ensuring that `SDL_SetCursor` is only called when absolutely necessary.

(As-is, if `wzSetCursor` is called twice in a main loop iteration - once to switch the cursor and a second time by other code to effectively switch it "back" - `SDL_SetCursor` may be called when it doesn't need to be.)

Reply:
Makes sense. Will fix